src/js/routes.js

```diff
@@ -579,6 +579,15 @@ angular.module('copayApp').config(function(historicLogProvider, $provide, $logPr
           }
         }
       })
+      .state('tabs.preferences.preferencesExternal', {
```

Comment:
Would be nice to add L&T information there, but for now maybe a conditional to hide it will avoid the L&T users confusing or asking "why the information is not available if I have my hardware connected" (like me lol)